LocalSettings.php

```diff
@@ -4203,6 +4203,16 @@
 			'bot' => array(
 				'editinterface' => true,
 			),
+			'user' => array(
```

Comment:
Except chat these are all default user permissions, so again they don't need to be included